csharp/ql/src/semmle/code/csharp/security/dataflow/UnsafeDeserialization.qll

```diff
@@ -68,11 +68,81 @@ module UnsafeDeserialization {
     override predicate isSanitizer(DataFlow::Node node) { node instanceof Sanitizer }
   }
 
+  /**
+   * User input to `JsonConvert` call deserialization flow tracking.
+   */
+  class JsonConvertTrackingConfig extends TaintTracking::Configuration {
```

Comment:
This configuration appears to be unused?

Reply:
It is used https://github.com/github/codeql/pull/5508/files#diff-a45002cff97b29b2d80a8c0d9e34dbd01b31dcf20592730a6f613f509e941091R52

Reply:
So it is, sorry.